src/scaffoldfitter/fitter.py

```diff
@@ -140,6 +142,10 @@ def decodeSettingsJSON(self, s: str, decoder):
         # ensure there is a first config step:
         if (len(self._fitterSteps) > 0) and isinstance(self._fitterSteps[0], FitterStepConfig):
             # field names are read (default to None), fields are found on load
+            id = settings.get("id")
+            if id is not None:
+                assert id == self.SCAFFOLD_FITTER_SETTINGS_ID
+                assert settings['version'] == '1.0.0'  # future: migrate if version changes
```

Comment:
This will probably have to evolve into a function in the future when changes are made to the current standard.